woocommerce/sensei-media-attachments
# Add gulp package tasks

Add gulp tasks to put together the release package.

## Testing

- Create the build package using `gulp package`

- Try uploading the built zip file to a WordPress installation

- Test Media Attachments to ensure that it works
